src/Symfony/Component/Translation/composer.json

```diff
@@ -21,6 +21,7 @@
     },
     "require-dev": {
         "symfony/config": "~2.8|~3.0",
+        "symfony/dependency-injection": "~3.3",
```

Comment:
you need to add a conflict rule in order to avoid having errors when using 3.3 features (these passes use ones)